VideoPlayer.js

```diff
@@ -1302,8 +1317,8 @@ export default class VideoPlayer extends Component {
             />
             {this.renderError()}
             {this.renderLoader()}
-            {this.renderTopControls()}
-            {this.renderBottomControls()}
+            {!this.props.hideAllControls ? this.renderTopControls() : null}
+            {!this.props.hideAllControls ? this.renderBottomControls() : null}
```

Comment:
```js
{!this.props.hideAllControls && this.renderTopControls()}
{!this.props.hideAllControls  && this.renderBottomControls()}

Reply:
or: 
```js
{!this.props.hideAllControls && this.renderTopControls()}
{!this.props.hideAllControls && this.renderBottomControls()}

Reply:
I gotta get used the the && operator aha. Im just so used to using ? : . It's just more readable for me aha

Reply:
? : is more readable for you than &&, thats weird lol

Reply:
Ahaha yeah its all good, Ill start doing && operator